akgraph/client/setup.py

```diff
@@ -1,27 +1,27 @@
-from setuptools import setup, find_packages
 from os import path
 
+from setuptools import find_packages, setup
 
 HERE = path.abspath(path.dirname(__file__))
-with open(path.join(HERE, 'README.md'), encoding='utf-8') as fh:
+with open(path.join(HERE, "README.md"), encoding="utf-8") as fh:
     LONG_DESCRIPTION = fh.read()
 
-setuptools.setup(
-    name='akgraph',
-    version='0.2.0',
+setup(
+    name="akgraph",
+    version="0.2.0",
     author="U.S. Government",
     author_email="",
     description="Graph algorithms in arkouda",
     long_description=LONG_DESCRIPTION,
     long_decription_content_type="text/markdown",
     keywords="HPC graph algorithm",
-    packages=setuptools.find_packages(),
+    packages=find_packages(),
     classifiers=["Programming Language :: Python :: 3"],
     use_scm_version=True,
-    setup_requires=['setuptools_scm'],
-    install_requires=['h5py','numpy'],
-    python_requires='>=3.6',
+    setup_requires=["setuptools_scm"],
+    install_requires=["h5py", "numpy"],
+    python_requires=">=3.8",
     project_urls={
-        'Bug Reports': 'https://github.com/Bears-R-Us/arkouda/issues',
-    }
+        "Bug Reports": "https://github.com/Bears-R-Us/arkouda/issues",
```

Comment:
should we change this to point to `https://github.com/Bears-R-Us/arkouda-contrib/issues`?

Reply:
yes

Reply:
@Ethan-DeBandi99 could you fix this so we can merge it? thanks

Reply:
This has been updated now. Sorry for the delay!